frontend/www/js/omegaup/contest/mine.ts

```diff
@@ -100,8 +105,12 @@ OmegaUp.on('ready', () => {
     },
   });
 
-  function fillContestsTable(showAllContests: boolean): void {
-    (showAllContests ? api.Contest.adminList() : api.Contest.myList())
+  function fillContestsTable(
+    showAllContests: boolean,
+    showArchivedContests: boolean,
+  ): void {
```

Comment:
```suggestion
  function fillContestsTable({
    showAllContests,
    showArchivedContests,
  }: {
    showAllContests: boolean,
    showArchivedContests: boolean,
  }): void {
```

en general si hay más de un parámetro del mismo tipo juntos, hay que preferir esta manera de recibir parámetros para evitar errores accidentales por transposición.

Reply:
Listo